### PR TITLE
feat: added product listing page

### PR DESCRIPTION
**Pull request by paarthbajaj**
product-listing

### CODE REVIEW

File: homepage.css [hunk 1]
```diff
@@ -10,7 +10,8 @@
 :root {
   --gray-shade1: #f5f5f6;
   --gray-shade2: #696e79;
-  --grey-shade3: #495057;
+  --gray-shade3: #495057;
+  --gray-shade4: #f1f3f6;
```
Comment (ozairkhan2810) [inline homepage.css:14]: numbers are not understandable for colors, you can use like dark, darker, shade, light, etc

File: homepage.css [hunk 2]
```diff
@@ -10,7 +10,8 @@
...
   --transaparent-grey: #c7c7c7e5;
```
Comment (ozairkhan2810) [inline homepage.css:16]: this should be #c7c7c7 i think or correct me

File: product-listing.css [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+.card-img,
+.img-wrap {
+  height: 17rem;
+  width: 17rem;
+  object-fit: cover;
+}
+.img-wrap {
+  position: absolute;
+  z-index: 1;
+  background: rgb(237, 237, 237);
```
Comment (ozairkhan2810) [inline product-listing.css]: use color variable

File: product-listing.css [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+  z-index: 2;
+}
+.pd-wish-icon {
+  position: absolute;
+  z-index: 2;
+  top: 0.4rem;
+  right: 0.4rem;
+  color: var(--gray-shade4);
+  font-size: 1.4rem;
+  cursor: pointer;
```
Comment (ozairkhan2810) [inline product-listing.css]: make cursor: pointer utility class as you are using it many times, then you can directly place in html

**Review by ozairkhan2810 — APPROVED** [review]
suggested for refactor, looks good to merge

File: homepage.css (hunk 1 above)
Comment (paarthbajaj) [reply homepage.css:14]: Actually, I have seen it in bigger companies like Flipkart, etc. So I thought they must have done it in the right way.

File: homepage.css (hunk 2 above)
Comment (paarthbajaj) [reply homepage.css:16]: No the color code if of grey with less opacity.

File: product-listing.css (hunk 3 above)
Comment (paarthbajaj) [reply product-listing.css]: I will correct it, thank you.

File: product-listing.css (hunk 4 above)
Comment (paarthbajaj) [reply product-listing.css]: Yeah, will do. Thank you.

File: product-listing.css [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+  position: absolute;
+  right: 0.4rem;
+  bottom: 5.5rem;
+  z-index: 2;
+}
+.pd-wish-icon {
+  position: absolute;
+  z-index: 2;
+  top: 0.4rem;
+  right: 0.4rem;
```
Comment (sks-21) [inline product-listing.css:63]: for values less than 10px, don't use rem

**Review by sks-21 — APPROVED** [review]
Rest everything looks good

File: product-listing.css (hunk 5 above)
Comment (paarthbajaj) [reply product-listing.css:63]: ok, noted. Thanks.

**Pull request merged**